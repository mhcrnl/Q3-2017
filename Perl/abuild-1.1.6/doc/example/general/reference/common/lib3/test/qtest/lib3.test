#!/usr/bin/env perl

require 5.008;
BEGIN { $^W = 1; }
use strict;

chdir("lib3") or die "chdir testdir failed: $!\n";

require TestDriver;

my $td = new TestDriver('lib3');

$td->runtest("test lib3 class",
	     {$td->COMMAND => "lib3_test"},
	     {$td->FILE => "lib3.out", $td->EXIT_STATUS => 0},
	     $td->NORMALIZE_NEWLINES);

$td->report(1);
