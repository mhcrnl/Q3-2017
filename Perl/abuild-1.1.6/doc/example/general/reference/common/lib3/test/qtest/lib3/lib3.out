not counting to -3
They say that -3 is an interesting number.
counting to 5: 1 2 3 4 5
They say that 5 is an interesting number.
